client/state/plugins/installed/test/selectors.js

```diff
@@ -17,20 +17,30 @@ const createError = function ( error, message, name = false ) {
 	return errorObj;
 };
 
+const userStateCopy = { ...userState };
+
+const siteOneId = 12345;
+const siteTwoId = 54321;
+const siteThreeId = 21435;
+
+userStateCopy.currentUser.capabilities[ siteOneId ] = { manage_options: true };
+userStateCopy.currentUser.capabilities[ siteTwoId ] = { manage_options: true };
+userStateCopy.currentUser.capabilities[ siteThreeId ] = { manage_options: true };
+
 const state = deepFreeze( {
 	plugins: {
 		installed: {
 			isRequesting: {
-				'site.one': false,
```

Comment:
The site IDs in production are numbers and not strings so this was causing some tests to fail due to type mismatches. Making these IDs numbers fixed that.